cmd/autoowners/main.go

```diff
@@ -380,6 +386,11 @@ func listUpdatedDirectoriesFromGitStatusOutput(s string) ([]string, error) {
 	return directories, nil
 }
 
+type blocklist struct {
+	directories sets.String
```

Comment:
since this represents a repository, let's call it `repo`

Reply:
No, it's actually directories, not repos... See how it is used: 

https://github.com/openshift/release/blob/94291a3c8fab4ad6eede56341f56f0d7d4965e1e/ci-operator/jobs/infra-periodics.yaml#L635-L670

Reply:
(I was also quite surprised, I originally called this `repositories`, too)